test/server/listener_manager_impl_test.cc

```diff
@@ -320,6 +333,7 @@ TEST_F(ListenerManagerImplTest, AddListenerAddressNotMatching) {
   ListenerHandle* listener_foo = expectListenerCreate(false);
   EXPECT_CALL(listener_factory_, createListenSocket(_, true));
   EXPECT_TRUE(manager_->addOrUpdateListener(*loader));
+  checkStats(1, 0, 0, 0, 1, 0);
```

Comment:
I'm guilty of this as well (in the Subscription stats PR), but it's actually a bit unreadable these `checkStats` statements, since you don't know what each magic number means. It might be nicer to be more verbose and pass in a struct with named fields , e.g. `{.warming = 1, .active = 0, ...}` or the like. Up to you if you want to change anything here, I don't feel that strongly.